lldb/test/CMakeLists.txt

```diff
@@ -244,6 +244,7 @@ llvm_canonicalize_cmake_booleans(
   LLVM_ENABLE_ZLIB
   LLVM_ENABLE_SHARED_LIBS
   LLDB_HAS_LIBCXX
+  LLDB_SHELL_TESTS_DISABLE_REMOTE
```

Comment:
It would be good to declare this and the other new options in CMake with docstrings so that they show up in tools like ccmake and just generally have another way to be discovered. `test/API/CMakeLists.txt` does this for example:
```
set(LLDB_TEST_COMPILER "${LLDB_DEFAULT_TEST_COMPILER}" CACHE PATH "C Compiler to use for building LLDB test inferiors")
```

Reply:
Done